objcgen/driver.cs

```diff
@@ -521,6 +521,7 @@ public int Compile ()
 						mtouch.Append ("--msym:false ");
 						mtouch.Append ("--nosymbolstrip ");
 						mtouch.Append ($"--embeddinator ");
+						mtouch.Append ($"--xml={Path.Combine (OutputDirectory, "linker.xml")} ");
```

Comment:
The idea was to generate the XML file depending on the assemblies code, e.g. `System.Decimal` is not always needed (it's even uncommon) so we don't want all project to pay the price for it.

However this can be used until the correct support is added, just add a `FIXME` until then.